easybuild/easyconfigs/o/OpenStackClient/OpenStackClient-6.0.0-GCCcore-12.2.0.eb

```
@@ -0,0 +1,129 @@
+easyblock = 'PythonBundle'
+
+name = 'OpenStackClient'
+version = '6.0.0'
+
+homepage = 'https://docs.openstack.org/python-openstackclient/latest/'
+description = """OpenStackClient (aka OSC) is a command-line client for OpenStack that brings the
+command set for Compute, Identity, Image, Network, Object Store and Block       
+Storage APIs together in a single shell with a uniform command structure."""
+
+toolchain = {'name': 'GCCcore', 'version': '12.2.0'}
+dependencies = [
+    ('Python', '3.10.8'),
+    ('PyYAML', '6.0'),
+]
+builddependencies = [('binutils', '2.39')]
+
+exts_default_options = {
+    'use_pip': True,
+}
+exts_list = [
+    ('pyperclip', '1.8.2', {
+        'checksums': ['105254a8b04934f0bc84e9c24eb360a591aaf6535c9def5f29d92af107a9bf57'],
+    }),
+    ('importlib_metadata', '5.0.0', {
+        'checksums': ['da31db32b304314d044d3c12c79bd59e307889b287ad12ff387b3500835fc2ab'],
+    }),
```

```suggestion
    ('importlib_metadata', '5.0.0', {
        'checksums': ['da31db32b304314d044d3c12c79bd59e307889b287ad12ff387b3500835fc2ab'],
    }),
```

https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/p/Python/Python-3.10.8-GCCcore-12.2.0.eb#L191

---

Oh, I didn't noticed this. Thank you! Done in https://github.com/easybuilders/easybuild-easyconfigs/pull/16677/commits/cc7a26bef21b44ae8a6ce40df5f02f6b81e23a13